content/blog/2020-09-22-introducing-the-new-jsx-transform.md

```diff
@@ -54,7 +54,7 @@ function App() {
 
 However, this is not perfect:
 
-* Because JSX compiled into `React.createElement`, `React` needed to be in scope if you use JSX.
+* Because JSX was compiled into `React.createElement`, `React` needed to be in scope if you use JSX.
```

Comment:
```suggestion
* Because JSX was compiled into `React.createElement`, `React` needed to be in scope if you used JSX.
```

English is only my second language but shouldn't this sentence either be in past or present but not in a mixed state?

Reply:
You're right! Just changed it :)